js/extension/cadastrapp.css

```diff
@@ -3,6 +3,11 @@
   white-space: normal !important;
 }
 
+.cadastrapp, .cadastrapp-modal {
+  /* update color palette to align with the custom theme variables of geOrchestra (https://github.com/georchestra/mapstore2-georchestra/blob/master/themes/default/variables.less) */
+  --ms-primary: #85127e;
```

Comment:
I'm afraid that defining this value will not use the theme, but only the value defined here.
So if an installation have a different theme, it will not follow the theme.

Reply:
Yes, it won't use the value defined in the container app. I thought this was a limitation of the extension in applying the container app's theme, as it includes custom CSS to override styles (please correct me if I'm mistaken). Since Cadastrapp applied a custom color code, I merely updated it to align with geOrchestra's and added a comment to indicate this

Reply:
Correct me if I'm wrong, I mean :
- If you don't define the variable here, the variable should be anyway defined in the main theme, and so inherit from it.
- If you define it, like you did, it will always use that color, ignoring the theme

If so, and if we don't have any other problem (like building issues or so) we should avoid to define the variable value here, letting the mapstore theme to define it.

You can test it by changing the variable in the test enviornment theme and installing the extension in 2 different contexts, with different themes.